src/singletons/Logging.cpp

```diff
@@ -49,27 +46,36 @@ void Logging::addMessage(const QString &channelName, MessagePtr message,
         }
     }
 
-    auto platIt = this->loggingChannels_.find(platformName);
-    if (platIt == this->loggingChannels_.end())
+    this->loggingChannels_.at(platformName)
+        .at(channelName)
+        ->addMessage(message);
```

Comment:
This feels dangerous. Although all channels initialize themselves, it would be nice if this didn't throw if the platform or the channel can't be found.

Reply:
throw should never happen, because addMessage is called only after loggingChannels_ was populated with same set of args, i would argue it failing instantly is better than hiding potential problems